src/app/home/recent-spaces-widget/recent-spaces-widget.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="card-pf {{cardSizeClass}}">
+  <div class="card-pf-heading f8-card-heading">
+    <div class="card-pf-heading-details f8-card-heading-details">
+      <a class="btn btn-link f8-card-heading-btn-link" (click)="showAddSpaceOverlay()">
+        <i class="pficon pficon-add-circle-o"></i>
+      </a>
+    </div>
+    <h2 class="card-pf-title">
+      Recent Spaces
+    </h2>
+  </div>
+  <div class="card-pf-body f8-card-body">
```

Comment:
While you're here, the widget seems to have the f8-card-body height of 375px [[0]](https://github.com/fabric8-ui/fabric8-ui/blob/master/src/assets/stylesheets/shared/_cards.less#L17) regardless of if it's large or not. 

Example: here's a exp/internal flagged large f8-card, but since the height is maxed out at 375px it doesn't fill the component
![recent-spaces-cutoff](https://user-images.githubusercontent.com/10425301/43226251-54880eda-9029-11e8-9c9d-c884f772ef12.png)

[0] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/assets/stylesheets/shared/_cards.less#L17

Reply:
I think the large f8-card needs a different (and larger) height dimension. Or potentially just fill the component, the spacesservice.recent only returns at max 8 spaces which isn't enough to completely fill the large card and cause the scrollbar anyways.

Reply:
Ah, so it's the combination of f8-card-lg and f8-card-body that isn't working together well? Is this something that should be fixed in the shared CSS then?

Reply:
Yeah, so for the exp/internal you're passing the f8-card-lg to the recent-spaces component, so that the entire component becomes a large card [0]. However, the body of the card is still a f8-card-body [1], and there isn't a distinct CSS rule for displaying large cards. I'm thinking you could probably nest a `&-body` to match `f8-card-lg-body` under the f8-card rule [2] to adjust for the size, and then have the card body [1] be `{{cardSizeClass}}`.

[0] https://github.com/andrewazores/fabric8-ui/blob/d85f39ddc234e94aab8365d5124eb166c004722c/src/app/home/recent-spaces-widget/recent-spaces-widget.component.html#L1
[1] https://github.com/andrewazores/fabric8-ui/blob/d85f39ddc234e94aab8365d5124eb166c004722c/src/app/home/recent-spaces-widget/recent-spaces-widget.component.html#L12
[2] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/assets/stylesheets/shared/_cards.less#L10

Reply:
I see what you mean, thanks. Does nesting it as `f8-card-lg-body` make sense though? Seems like it goes against the naming convention, which would have it as `f8-card-body-lg`. This makes it a little messier to specify the sizing of the component since now there are two inputs that are passed in, but that's only a temporary workaround to deal with feature flagging anyway.